src/components/social-media-component/assets/FacebookIcon.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
All icons should be stored in the same place so move the file to .src/icons, make it use also IconProps, remove <a> tag and use onClick with window.location.href